commerce-frontend-taglib/src/main/resources/META-INF/resources/search_bar/SearchBar.es.js

```diff
@@ -75,7 +75,7 @@ class SearchBar extends Component {
 
 		this.active = status;
 
-		this.emit('toggled', status);
+		this.emit('toogled', status);
```

Comment:
Is there any way we can just change this to `toggled` everywhere? `toogled` isn't a thing :(